src/main/resources/templates/search.html

```diff
@@ -38,6 +38,34 @@ <h2>Search by:</h2>
 
     <!-- TODO #4 - Loop over the search results to display all job fields. -->
 
+    <table class="job-listing" th:each="job: ${jobs}">
+        <tr>
+            <td>ID</td>
+            <td th:text="${job.getId()}"></td>
```

Comment:
```suggestion
            <td th:text="${job.id}"></td>
```
You can use the field names of `job` instead of the getter methods to display the values.